### PR TITLE
Bug 1420225 - Read legacy files when scanning for Sources in transforms

### PR DESCRIPTION
**Pull request by stasm**
https://bugzilla.mozilla.org/show_bug.cgi?id=1420225

This is a simple change which makes it unnecessary to use `maybe_add_localization`—at all.

### CODE REVIEW

File: fluent/migrate/context.py [hunk 1]
```diff
@@ -139,7 +139,7 @@ def maybe_add_localization(self, path):
         else:
             self.localization_resources[path] = collection
 
-    def add_transforms(self, path, reference, transforms):
+    def add_transforms(self, target, reference, transforms):
         """Define transforms for path using reference as template.
```
Comment (Pike) [inline fluent/migrate/context.py]: Want to adjust the comment to change from path to target, too?

Comment (stasm) [reply fluent/migrate/context.py]: Sure, I considered it but thought that the current comment was still OK. Happy to change it.

Comment (Pike) [reply fluent/migrate/context.py]: I think an udpate would be good, as right now, `for path` isn't good English, if path isn't the argument.

Comment (stasm) [reply fluent/migrate/context.py]: Yeah, agreed. I updated the comment and also added two sentences about what those paths are supposed to be relative to.

**Pull request merged**